draft-gazdag-x509-shbs.md

```diff
@@ -112,81 +104,76 @@ informative:
 --- abstract
 
 This document specifies algorithm identifiers and ASN.1 encoding formats for
-the Hash-Based Signature (HBS) schemes Hierarchical Signature System (HSS),
-eXtended Merkle Signature Scheme (XMSS), and XMSS^MT, a multi-tree variant of
-XMSS, as well as SLH-DSA (formerly SPHINCS+), the latter being the only
-stateless scheme. This specification applies to the Internet X.509 Public Key
+the Stateful Hash-Based Signature (S-HBS) schemes Hierarchical Signature System
+(HSS), eXtended Merkle Signature Scheme (XMSS), and XMSS^MT, a multi-tree
+variant of XMSS. This specification applies to the Internet X.509 Public Key
 infrastructure (PKI) when those digital signatures are used in Internet X.509
 certificates and certificate revocation lists.
 
 --- middle
 
 # Introduction
 
-Hash-Based Signature (HBS) Schemes combine Merkle trees with One/Few Time
-Signatures (OTS/FTS) in order to provide digital signature schemes that remain
-secure even when quantum computers become available. Their theoretic security
-is well understood and depends only on the security of the underlying hash
-function. As such they can serve as an important building block for quantum
-computer resistant information and communication technology.
-
-The private key of HSS, XMSS and XMSS^MT is a finite collection of OTS keys,
-hence only a limited number of messages can be signed and the private key's
-state must be updated and persisted after signing to prevent reuse of OTS keys.
-While the right selection of algorithm parameters would allow a private key to
-sign a virtually unbounded number of messages (e.g. 2^60), this is at the cost
-of a larger signature size and longer signing time. Due to the statefulness of
-the private key of HSS, XMSS and XMSS^MT and the limited number of signatures
-that can be created, these signature algorithms might not be appropriate for
-use in interactive protocols. However, in some use case scenarios the
-deployment of these signature algorithms may be appropriate. Such use cases are
-described and discussed later in {{use-cases-hbs-x509}}.
-
-The private key of SLH-DSA is a finite but very large collection of FTS keys
-and hence stateless. This typically comes at the cost of larger signatures
-compared to the stateful HBS variants. Thus SLH-DSA is suitable for more use
-cases if the signature sizes fit the requirements.
+The Stateful Hash-Based Signature (S-HBS) schemes HSS, XMSS and XMSS^MT combine
```

Comment:
Reading this implies that S-HBS = Stateful Hash-Based Signature. However, I would suggest to define 
S-HBS = Stateful Hash-Based Signature schemes